spec.html

```diff
@@ -358,10 +358,15 @@ <h3>Notes</h3>
       type uses delta and delta-of-delta compression and
       run-length-encoding for efficient element storage. Also has an
       encoding for sparse arrays containing missing values.</li>
-    <li>Vector - A densely packed array of numbers, all of the same
-      type. This subtype supports the packed
-      binary (1-bit unsigned <code>int</code>), signed 8-bit
-      <code>int</code>, and 32-bit <code>float</code> element types.
+    <li>Vector - A densely packed array of numeric values. All values must be of the same type.
```

Comment:
I think we should rework the opening sentence. I suggest we use a more standard definition of vectors in programming, maybe something like:

> Array of numeric values that can be dynamically resized. All values must be of the same type. Vectors support the following types: 
>
>  (bulleted list of types)
>
> Vectors are useful for bitmaps, machine learning, scientific computations, or other use cases that require optimized representation of numeric values.